### PR TITLE
[IMP] website_apps_store: improved download button

### PR DESCRIPTION
**Pull request by oscarolar**
**Before**
image

**After**
image

### CODE REVIEW

**Comment by oscarolar** [comment]
cc @elicoidal

File: website_apps_store/views/templates.xml [hunk 1]
```diff
@@ -138,8 +138,8 @@
             </xpath>
             <xpath expr='//form[hasclass("js_add_cart_variants")]' position="after">
                 <div class="col-xs-12 padd_0" t-if="product.odoo_module_id">
-                    <div class="col-xs-4 zip_button">
-                        <button name="download_zip" id="download_zip" t-att-data-tmpl-id="product.id">Download</button>
+                    <div class="col-xs-4 mt16 mb16">
+                        <button class="btn btn-primary btn-lg" name="download_zip" id="download_zip" t-att-data-tmpl-id="product.id">Download <span class="fa fa-download"/></button>
```
Comment (elicoidal) [inline website_apps_store/views/templates.xml]: Please improve the format for readability purpose.

**Review by elicoidal — APPROVED** [review]
Version number.
LGTM

File: website_apps_store/views/templates.xml (hunk 1 above)
Comment (oscarolar) [reply website_apps_store/views/templates.xml]: @elicoidal done

**Review by RoelAdriaans — APPROVED** [review]

**Comment by bizzappdev** [comment]
@oscarolar  we also have to take care for having HTML description of the Module in the container.
it was your suggestion only.

**Pull request merged**